examples/daal/cpp/source/decision_forest/df_cls_default_dense_batch.cpp

```diff
@@ -85,8 +90,15 @@ training::ResultPtr trainModel() {
     algorithm.parameter().minWeightFractionInLeafNode = minWeightFractionInLeafNode;
     algorithm.parameter().minImpurityDecreaseInSplitNode = minImpurityDecreaseInSplitNode;
     algorithm.parameter().varImportance = algorithms::decision_forest::training::MDI;
-    algorithm.parameter().resultsToCompute =
-        algorithms::decision_forest::training::computeOutOfBagError;
+    if (extraTrees) {
+        /* Extra Trees is set through two parameters which are different by default */
+        algorithm.parameter().bootstrap = false;
+        algorithm.parameter().splitter = algorithms::decision_forest::training::random;
+    }
+    else {
+        algorithm.parameter().resultsToCompute =
+            algorithms::decision_forest::training::computeOutOfBagError;
+    }
```

Comment:
Please just move ExtraTrees examples into a different file

Reply:
@Alexsandruss, I now have two others (including @Alexandr-Solovev ) who have recommended separate files, is there a standard for this or should we make one? I'd side with Nikita on this, and I've reverted my changes back to what he recommends.

Reply:
@icfaust Examples should be simple as much as possible and for separate algo separate one.